Actions/CheckForUpdates/CheckForUpdates.HelperFunctions.ps1

```diff
@@ -251,7 +295,7 @@ function GetWorkflowContentWithChangesFromSettings {
     # PullRequestHandler, CICD, Current, NextMinor and NextMajor workflows all include a build step.
     # If the dependency depth is higher than 1, we need to add multiple dependent build jobs to the workflow
     if ($depth -gt 1 -and ($baseName -eq 'PullRequestHandler' -or $baseName -eq 'CICD' -or $baseName -eq 'Current' -or $baseName -eq 'NextMinor' -or $baseName -eq 'NextMajor')) {
-        ModifyBuildWorkflows -yaml $yaml -depth $depth
+        ModifyBuildWorkflows -yaml $yaml -depth $depth -includeBuildPP ($repoSettings.type -eq 'PTE')

```

Comment:
We could do that - but this would mean that you would have to run Update AL-Go System Files after adding a PowerPlatformSolutionFolder. I guess this would be fine - or?
This would also remove the jobs in build etc when no PowerPlatformSolutionFolder exists

Reply:
To be honest.. I'm leaning a bit in the opposite direction. I think we PP Build step should always be there. I'm not sure how well this approach scales if we want to add support for other types of projects in the future. 

What's the downside of keeping it in? That there is a skipped build step?

Reply:
With the latest fix - the workflows and the build jobs are there when you create a repo, but they are removed if you don't have PowerPlatformSolutionFolder defined and run Update AL-Go System Files. And I added this note to the releasenotes:

> [!NOTE]
> PowerPlatform workflows are only available in the PTE template and will be removed if no PowerPlatformSolutionFolder is defined in settings.